src/ssl.c

```diff
@@ -48993,7 +48993,12 @@ WOLF_STACK_OF(WOLFSSL_CIPHER) *wolfSSL_get_ciphers_compat(const WOLFSSL *ssl)
 #if defined(OPENSSL_ALL) || defined(WOLFSSL_QT)
         int j;
 #endif
+#if defined(WOLFSSL_QT)
+        /* higher priority of cipher suite will be on top of stack */
+        for (i = suites->suiteSz - 2; i >=0; i-=2) {
```

Comment:
Is this Qt-specific, or just OpenSSL-specific.  If the later, should it be protected under OPENSSL_EXTRA/ALL instead of WOLFSSL_QT?

Reply:
It is correct. It should be protected by OPENSSL_EXTRA/ALL